bin/plugin/commands/build-plugins.js

```diff
@@ -0,0 +1,219 @@
+/**
+ * External dependencies
+ */
+const fs = require( 'fs-extra' );
+const path = require( 'path' );
+const glob = require( 'fast-glob' );
+
+/**
+ * Internal dependencies
+ */
+const { log, formats } = require( '../lib/logger' );
+
+exports.options = [];
+
+/**
+ * Command for copying the files required for the standalone plugin.
+ */
+exports.handler = async () => {
+	const pluginsFile = path.join( '.', 'plugins.json' );
+	fs.readFile( pluginsFile, 'utf8', ( err, jsonString ) => {
+		if ( err ) {
+			log(
+				formats.error(
+					`\nError reading file from disk: "${ err }"\n\n`
+				)
+			);
+		}
+
+		try {
+			const customer = JSON.parse( jsonString );
+			for ( const key in customer ) {
+				const pluginKey = key;
+				const pluginVersion = customer[ pluginKey ].version;
+				const pluginSlug = customer[ pluginKey ].slug;
```

Comment:
@mukeshpanchal27 (cc @joemcgill ) I don't want to nit pick at this too much given the context of when and how this command will be run, but we are assuming the presence of the `version` and `slug` object properties in all plugin objects in the json file.

While the reality is that they definitely _should always be present_ (and would/should be caught in PR's if not), the command would break the build flow for all plugins following any broken entry.

Do we need to consider checking the values of these properties before assuming their presence, or are we confident that this is such an "internal" usage of the commands that we're unlikely to ever see such an event occurring?

If I've missed some context that negates the above, please let me know.

Reply:
Thanks @10upsimon for review. Agreed added check in https://github.com/WordPress/performance/pull/662/commits/b19e57cd4f65a36829ddfa24cd1dad3f6771d60b